source/common/quic/udp_gso_batch_writer.h

```diff
@@ -18,8 +18,11 @@
 #pragma GCC diagnostic ignored "-Wsuggest-override"
 #endif
 
+#if defined(__linux__)
```

Comment:
Does this fix the build? There are other places in this file refers to quic classes.
Why not excluding this header file completely in the BUILD if the platform is not Linux?

Reply:
right, sorry, I had commented out most of the file in a prior build which got lost.

So I can add that back, moving the endif to the bottom of the file, or we could have one file for setting UDP_GSO_BATCH_WRITER_COMPILETIME_SUPPORT  and a second file for use of it.

which would you prefer?

Reply:
I'm okay either way, maybe the latter is easier to understand?